.cirrus.yml

```diff
@@ -0,0 +1,38 @@
+# By default, Cirrus mounts an empty volume to `/tmp`
+# which triggers all sorts of warnings like "system temporary path is world-writable: /tmp".
+# Lets workaround it by specifying a custom volume mount point.
+env:
+  CIRRUS_VOLUME: /cirrus
+
+task:
+  name: Test on Graviton2 ($CC)
+  arm_container:
+    image: ghcr.io/ruby/ruby-ci-image:$CC
+    cpu: 4
+    memory: 16G
+  env:
+    CIRRUS_CLONE_DEPTH: 0 # do a full clone since on a large history it's less CPU intense
+    RUBY_DEBUG: ci
+    NPROC: 4
+    JOBS: -j5
+    RUBY_TESTOPTS: "-q --tty=no"
+    matrix:
+      CC: clang-12
+      CC: gcc-11
+  setup_script:
+    - sudo apt-get update
+    - sudo apt-get install --no-install-recommends -q -y ruby
+    - sudo ln -snf /usr/share/zoneinfo/Japan /etc/localtime # set any timezone
+    - sudo dpkg-reconfigure --frontend noninteractive tzdata
+  disable_ipv6_script:
+    # Arm containers are executed in AWS's EKS, and it's not yet supporting IPv6
+    # See https://github.com/aws/containers-roadmap/issues/835
+    - sudo ./tool/disable_ipv6.sh
+  autogen_script: ./autogen.sh
+  configure_script: ./configure --disable-install-doc
+  incs_script: make $JOBS -s incs
+  make_all_script: make $JOBS -s all
+  test_script: make $JOBS -s test -o showflags TESTOPTS="$JOBS -q --tty=no"
```

Comment:
In the `.cirrus.yml`, about how to use the `RUBY_TESTOPTS`, there is a mixture of `.github/workflows/ubuntu.yml` and `.travis.yml`. As the `RUBY_TESTOPTS` is defined above in task - env, it's better to remove the part `TESTOPTS="$JOBS -q --tty=no"` on the command.

```
task:
  env:
    ...
    RUBY_TESTOPTS: "-q --tty=no"
 ```

Reply:
You don't need to update this line. I will update it by myself later. The 2 `$JOBS` variables are used in the the command. It's something redundant. We need to fix it.